apps/web/src/app/[locale]/(dashboard)/[wsId]/(workspace-settings)/migrations/components/ModuleSearch.tsx

```
@@ -0,0 +1,218 @@
+    }
+
+    // Note: We don't filter by empty data here since it's dynamic
+    // and will be handled by the parent component
+
+    // Apply sorting
+    result.sort((a, b) => {
+      switch (filterState.sortBy) {
+        case 'name':
+          return a.name.localeCompare(b.name);
+        case 'status':
+          const aStatus = a.disabled ? 2 : a.skip ? 1 : 0;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)